ql/src/test/queries/clientpositive/authorization_view_1.q

```diff
@@ -1,5 +1,6 @@
 --! qt:dataset:src
 set hive.security.authorization.manager=org.apache.hadoop.hive.ql.security.authorization.DefaultHiveAuthorizationProvider;
+set hive.exec.reducers.max=1;
```

Comment:
Any reason for changing this?

Reply:
yes, as this patch changes the behavior of all queries having limit, basically, all queries with >1 reducers are subject to intermittent result changes, because of making reducers exit early
that's what I experienced in early precommit runs, and I didn't want to introduce ORDER BY clauses